ui-pf4/src/main/webapp/package.json

```diff
@@ -45,7 +45,7 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts --max_old_space_size=2048 build",
```

Comment:
@mrizzi I don't think there is anything wrong with keeping the `--max_old_space_size=2048` in the package.json file... just one question, `react-scripts --max_old_space_size=2048 build` works but `react-scripts build --max_old_space_size=2048` doesn't? I would go for the second option because it is easier to read (for humans) but if the second option doesn't work then the first option should be ok too.

Reply:
Good point.
TBH I found it defined in this way in some issues just like this one https://github.com/facebook/create-react-app/issues/4536#issuecomment-393647911 and I didn't want "problems" with the option so I used it as suggested.